edits/pyflakes_lint.py

```diff
@@ -20,7 +25,8 @@
     messages.TwoStarredExpressions,
 )
 
-CUSTOM_VALID_FUNCTIONS = {
+# List of Patch functions  
+CUSTOM_VALID_FUNCTIONS = [
```

Comment:
Can you refactor this so there's only one list of custom valid functions? Just so changes are easier to make in the future